user_docs/en/changes.md

```diff
@@ -5,7 +5,7 @@
 ### Important notes
 
 ### New Features
-
+* Support for math in PDFs has been added. This works for formulas that have MathML in an Associated File. Tagged formulas are supported in newer versions of TeX/LaTeX (9288, @NSoiffer)
```

Comment:
Please preserve the new line, use new lines for sentences, and use a # before the issue number

```suggestion

* Support for math in PDFs has been added.
This works for formulas that have MathML in an Associated File.
Tagged formulas are supported in newer versions of TeX/LaTeX (#9288, @NSoiffer)
```